Dockerfile

```diff
@@ -157,17 +166,17 @@ RUN --mount=type=cache,target=/root/.cache/microdnf:rw \
 
 # need to upgrade pip and install wheel before installing grpcio, before installing tensorflow on aarch64
 # use caching to speed up multi-platform builds
+COPY requirements.txt requirements.txt
 ENV PIP_CACHE_DIR=/root/.cache/pip
 RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install --upgrade pip && \
     pip install wheel && \
     pip install grpcio && \
     # pin to 3.10.0 to avoid error: libhdf5.so: cannot open shared object file: No such file or directory \
     # if not version is set, it will install the 3.11.0 version which, seems that does not have the h5py dependencies \
     # for arm yet.
     pip install h5py==3.10.0 && \
     pip install tensorflow
```

Comment:
Replate all these `pip install` commands with a single `pip install -r requirements.txt` one and then I will be happy to approve.

Reply:
I agreed with @israel-hdez . However, if we replace this,  we should update the requirements.txt file when the version changes.

Reply:
right, I've forget to do this, I've added the requirements file inside the container but forgot to install from it